gradle/project.gradle.kts

```diff
@@ -235,6 +235,9 @@ tasks {
         group = main
         workingDir = workDir
         dependsOn(copyLibs, jdk, kotlinc, "jar")
+        doFirst {
```

Comment:
Why is this necessary?

Reply:
because startup otherwise errored for me when the log directory for the gc was missing